backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/notification/impl/NotificationGroupNameRepository.java

```diff
@@ -56,14 +58,40 @@ public Set<NotificationGroupName> getAll()
 		return getNotificationGroupInternalNamesById().values();
 	}
 
+	@Override
+	@Cached
+	@Nullable
+	public UserId getDeadletterUserId(final NotificationGroupName notificationGroupName)
```

Comment:
imho you shall re-think the overall caching of this repository

Reply:
re-thought.